lib/ricohapi/mstorage/client.rb

```diff
@@ -55,8 +57,24 @@ def download(media_id)
       # GET /media/{id}/meta, GET /media/{id}/meta/exif, GET /media/{id}/meta/gpano,
       # GET /media/{id}/meta/user, GET /media/{id}/meta/user/{key}
```

Comment:
As proposed directly, I think 62th, 67th and 72th lines are rather unnecessary because it's clear where to call each API in source.